frontend/src/routes/index.svelte

```diff
@@ -101,16 +100,12 @@
         search()
     }
 
-    const getFixedGenreValues = (genres: {}) => {
-        formattedGenres = genres;
-        return Object.keys(genres).map(function (key) {
-            return genres[key];
-        });
-    }
-
     onMount(async () => {
         const inputField = document.getElementById('input-field')
-        setTimeout(function () { inputField.select(); }, 100);
+        setTimeout(function () { inputField.select(); }, 20);
```

Comment:
What does this do, and is it needed? The site still loads without.

Reply:
To do inputField.select() it has to be inside a timeout. Don't ask me why, has to do with the site loading I think. Without it, input will not be selected when returning from details page

Reply:
Ah oki, so only the function is delayed.